azure/storage/handler/cloud_blob_client_handler.rb

```diff
@@ -0,0 +1,75 @@
+require_relative "base_handler"
+require_relative "../auto_generated/core_types"
+require_relative "../auto_generated/cloud_queue_client_types"
```

Comment:
Should not require `cloud_queue_client*` for blob service related handlers right?